joss_paper/paper.md

```diff
@@ -0,0 +1,119 @@
+---
+title: '`UnfoldMakie.jl`: EEG/ERP visualization package'
+
+tags:
+  - Julia
+  - Event-related potentials (ERP)
+  - Electroencephalography (EEG)
+  - Plotting
+  - Visualization
+  - Interactivity 
+  - ERP image
+  - Topoplot
+  - ERP grid
+  - Butterfly plot
+  
+  
+authors:
+  - name: Vladimir Mikheev
+    orcid: 0000-0002-4738-6655
+    equal-contrib: true
+    affiliation: 1
+  - name: Benedikt Ehinger
+    orcid: 0000-0002-6276-3332
+    equal-contrib: true
+    affiliation: "1, 2"
+affiliations:
+ - name: University of Stuttgart, Institute for Visualization and Interactive Systems, Germany
+   index: 1
+ - name: University of Stuttgart, Stuttgart Center for Simulation Science, Germany
+   index: 2
+
+date: 30 October 2024
+bibliography: paper.bib
+
+---
+
+
+# Statement of need
+
+`UnfoldMakie.jl` is a Julia package for visualization of electroencephalography (EEG) data, with a focus on event-related potentials (ERPs) and regression-ERPs (rERPs). `UnfoldMakie.jl` fills a gap as one of the only dedicated EEG visualization libraries and offers ease of use, customization, speed, and detailed documentation. It allows for both explorative analysis (zooming/panning) and export to publication-ready vector graphics. This is achieved via multiple backends of `Makie.jl` [@danisch2021makie]: vector graphics with `CairoMakie.jl`, GPU-accelerated interactive graphics with `GLMakie.jl` and the browser-based `WGLMakie,jl`. 
+
+In an earlier study [@mikheev2024art], we asked novice and expert practitioners for insights into their ERP visualization practices. The results of this survey were used to develop and improve `UnfoldMakie.jl`. Further, `UnfoldMakie.jl` is agnostic (independent) of any specific analysis framework, but it nicely accompanies the rERP analysis package `Unfold.jl` [@ehinger2019unfold].
+
+The target audience of this package is anyone working with EEG, MEG, ERP, iEEG or other time-series data.
+
+# Functionality
+`UnfoldMakie.jl` excels in various fields:
+
+- Focused. `UnfoldMakie.jl` focuses exclusively on visualizing (r)ERPs, unlike other toolboxes such as `EEGLAB`, `FieldTrip`, `Brainstorm`, or `MNE`. This makes it easier to understand, use, and maintain the package in the future.
+
+- Customizable. The majority of EEG researchers perceive the flexibility of coding as the most important feature of the EEG toolbox [@mikheev2024art]. Consequently, users `UnfoldMakie.jl` have great flexibility in customizing most aspects of the plots, such as colors, line styles, and axis decorations. 
```

Comment:
should be "Consequently, users **of** `UnfoldMakie.jl` have great flexibility in customizing most aspects of the plots, such as colors, line styles, and axis decorations."